include/simclist.h

```diff
@@ -0,0 +1,971 @@
+/*
```

Comment:
This file should not be in the `include` directory - it's only needed internally. Belongs in `src` probably.

Reply:
Moved!